config/exclusions/php.yaml

```diff
@@ -0,0 +1,5 @@
+exclusions:
+  - id: Exclusions.Test
+    name: Exclude test source code
+    patterns:
+      - "(.*/Tests/.*)|/.*Test.php"
```

Comment:
If the `Tests` folder is present at the root level like `src` remove the `/` as the path in relative in CPG

Reply:
Yes, good point. I found examples of `tests` and `src` in project root